### PR TITLE
add API to query DOI.org

### PR DESCRIPTION
**Pull request by hmenager**
## Description
Add an endpoint to query the DOI (https://dx.doi.org) endpoint to retrieve information in the JSON format

#### Changelogs

* Add an endpoint to query the DOI (https://dx.doi.org) endpoint to retrieve information in the JSON format

## Definition of Done

* [x]  New lines are tested with unit tests (mandatory for the addition of a new parser: [More info](https://dabeplech.readthedocs.io/en/latest/contribution_guide/parser.html))
* [X]  Documentation has been updated (When adding new API connector: [More info](https://dabeplech.readthedocs.io/en/latest/contribution_guide/docs.html))
* [x]  Pull Request has been revised by a maintainer

### CODE REVIEW

File: docs/api_docs/api_services.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ KEGG
     :show-inheritance:
 
 
-TogoWS
+TogoS
```
Comment (khillion) [inline docs/api_docs/api_services.rst:15]: Who ate the "W"?

**Review by khillion — COMMENTED** [review]
Cool I did not know we could retrieve JSON from doi too!

This seems all good to me, only the letter which disappear from the documentation for TogoWS.

File: docs/api_docs/api_services.rst (hunk 1 above)
Comment (hmenager) [reply docs/api_docs/api_services.rst:15]: I did, sorry :flushed:

Comment (hmenager) [reply docs/api_docs/api_services.rst]: fixed

**Review by khillion — APPROVED** [review]

**Comment by khillion** [comment]
Thanks for the contribution!

**Pull request merged**